recipe/build.sh

```diff
@@ -12,6 +12,8 @@ cmake ${CMAKE_ARGS}\
   -DMI_BUILD_SHARED=ON \
   -DMI_BUILD_STATIC=OFF \
   -DMI_BUILD_OBJECT=OFF \
+  -DMI_OVERRIDE=OFF \
```

Comment:
I think this also changes the build significantly.

Must dedicated builds for `MI_OVERRIDE=ON` and `MI_OVERRIDE=OFF` exist?

Also is it possible to combine `track_feature` s to be able to manage the all of the relevant build options combinations?

Reply:
In `jemalloc` we have two different packages `jemalloc` and `jemalloc-local` to enable both uses. But in the `jemalloc` case you can rename the library so that both can co-exist. Here it is probably not possible.